app/src/main/java/com/keylesspalace/tusky/util/LinkHelper.java

```diff
@@ -172,8 +171,11 @@ public static void openLinkInBrowser(Uri uri, Context context) {
      * @param context context
      */
     public static void openLinkInCustomTab(Uri uri, Context context) {
-        boolean lightTheme = PreferenceManager.getDefaultSharedPreferences(context).getBoolean("lightTheme", false);
-        int toolbarColor = ContextCompat.getColor(context, lightTheme ? R.color.custom_tab_toolbar_light : R.color.custom_tab_toolbar_dark);
+        TypedValue toolbarColorTv = new TypedValue();
+        context.getTheme().resolveAttribute(
+                context.getResources().getIdentifier("custom_tab_toolbar", "attr", context.getPackageName()), toolbarColorTv, true);
+
+        int toolbarColor = toolbarColorTv.data;
```

Comment:
there should already be a helper method for this in com.keylesspalace.tusky.util.ThemeUtils